haystack/document_store/elasticsearch.py

```diff
@@ -1053,6 +1100,9 @@ def _create_document_index(self, index_name: str):
             if not self.client.indices.exists(index=index_name):
                 raise e
 
+    def stop_service(self):
```

Comment:
Would it make sense to keep this in the utils instead? It can be misleading as it only works in the case when OpenSearch is running as a Docker Container running locally.